examples/tutorials/data/veritas.py

```diff
@@ -108,7 +108,7 @@
 # method.
 #
 
-observations[0].peek()
+observations[0].peek(figsize=(14,4))
```

Comment:
```suggestion
observations[0].peek(figsize=(14,4))
plt.show()
```

We also need to add a `plt.show()` here